File: LeetCode/JavaScript/#283 Move Zeroes.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var moveZeroes = function(nums) {
    var len = nums.length;
    var count = 0;
    var pos;
    while(nums.indexOf(0)!==-1&&count!==len){
      pos = nums.indexOf(0);
      nums[len] = 0;
      nums.splice(pos,1);
      count++;
    }
};

File: LeetCode/Python/#283 Move Zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        j = 0
        lens = len(nums)
        for i in range(0,lens):
            if(nums[i]!= 0):
                nums[j] = nums[i]
                j+=1

        for k in range(j,lens):
            nums[k] = 0